ui/imports/shared/status/StatusChatImageLoader.qml

```diff
@@ -10,7 +10,8 @@ Item {
     property int imageWidth: 350
     property bool isCurrentUser: false
     property url source
-    property bool playing: Global.applicationWindow.active
+    property bool isActiveChannel: true
+    property bool playing: Global.applicationWindow.active && isChatActive
```

Comment:
`readonly` for the latter?

Reply:
> isChatActive is coming from? :)

It comes from here https://github.com/status-im/status-desktop/pull/6290/files#diff-c02909c050778d76fd6f50eb86f983737ad65d485d17f71dc2daa6e7e2954392R550
StatusChatImage is a wrapper around StatusChatImageLoader.

> readonly for the latter?

I tried setting it readonly and it doesn't work, because some components set it manually at `false` to make sure that we don't animate.

Reply:
Ok, thanks, I see